source/agora/node/Ledger.d

```diff
@@ -427,7 +436,7 @@ public class Ledger
 
     ***************************************************************************/
 
-    protected void updateValidatorSet (const ref Block block) @safe
+    public void updateValidatorSet (const ref Block block) @safe
```

Comment:
This was made public, but I don't see it being called anywhere new.

Reply:
Reverted this change.